doc/source/ray-core/placement-group.rst

```diff
@@ -146,28 +151,20 @@ Strategy types
 
 Ray currently supports the following placement group strategies:
 
-**STRICT_PACK**
-
-All bundles must be placed into a single node on the cluster.
+**PACK**: All provided bundles are packed onto a single node on a best-effort basis.
```

Comment:
I might be missing some context with this. I'm not certain what "best-effort basis" is in this case. I would suggest rewording unless it's a known thing.